server/webapp/WEB-INF/urlrewrite.xml

```diff
@@ -23,6 +23,12 @@
     http://tuckey.org/urlrewrite/
 -->
 <urlrewrite>
+    <rule>
```

Comment:
One webapp context handler is up, `/go/` and `/go/home` come here and will be redirected to `/go/pipelines`. No way to support `SystemEnvironment.landingPage()`.

Reply:
The URL `http://localhost:8253/gO/` gives `404`

Reply:
I think it is better if we can handle it https://github.com/gocd/gocd/pull/4874/files#diff-fe8496d349656d6b77e18b8dce369db4R64. That way we don't have to rewrite url's. WDYT?

Reply:
@bdpiparva I responded to /gO just now in a different comment. About handling this in the GoServerLoadingIndicationHandler: It's not possible. This is what the different handlers claim:

```
GoServerLoadingIndicationHandler => /
WebAppContext                    => /go
AssetsHandler                    => /go/assets (I think)
```

Once all the handlers are up, any request to `/go/...` (like `/go/home`) will directly go to `WebAppContext` (because of [this](https://github.com/eclipse/jetty.project/blob/jetty-9.4.8.v20171121/jetty-server/src/main/java/org/eclipse/jetty/server/handler/ContextHandlerCollection.java#L208)). So, WebAppContext handler will get that call and not the loading handler.

This is because we moved to a ContextHandlerCollection (see the changes in Jetty9Server).